src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -80,6 +88,14 @@ def time_range(self) -> Tuple[int, int]:
     def _cache_path(self, events_path: str) -> str:
         return os.path.join(events_path, 'caret_converted')
 
+    @staticmethod
+    def _trace_dir_exists(path: str) -> bool:
+        return os.path.exists(path)
+
+    @staticmethod
+    def _cache_exists(path: str) -> bool:
+        return os.path.exists(path)
```

Comment:
> I will summarize this function into one function like file_exists() function.

Yes. These functions are intendedly separeted for tests.

Reply:
I've wrote docstrings in [49f7bd1](https://github.com/tier4/CARET_analyze/pull/189/commits/49f7bd15c620a8237bcdd1ff0b4c5d53c0d560d5) because this intent was not clear.